scripts/requirements-test.yml

```diff
@@ -41,6 +41,7 @@ dependencies:
   - h5py
   - pillow
   - libgcc-ng
+  - libstdcxx-ng
```

Comment:
found this in `create_local_env_for_cil_development.sh`... is it needed?

Reply:
also the `libgcc-ng` should be `# [linux]`